### DISCUSSION TITLE
Can I run Carla on an ARM based ubuntu?

### DISCUSSION THREAD

**Discussion by josephmsaed** [carla-simulator/carla#7321] [General]
I am using a virtualised arm ubuntu on my M2 macbook? would Carla work there? if not is there a better solution?

**Comment by xavisolesoft** [comment]
Unfortunately we are not building CARLA officially for ARM, so we do not have a Release that could work for your proposed configuration.

ARM build seems an interesting topic to be opened as issue for future work as most of the modern MacBooks use ARM as processor.

**Reply by josephmsaed** [reply]
Alright thank you, indeed ARM build would be an interesting future work, since the adoption rate of this architecture is currently increasing significantly.

**Reply by mylegitches** [reply]
I have great interest in an ARM build.

**Comment by nveshaan** [comment]
take a look at https://github.com/carla-simulator/carla/discussions/9037

**Comment by 10084743-sketch** [comment]
HOW DO I GET STARTED? [10000 likes]
